styleguide/components/21-atoms/link/_link.scss

```diff
@@ -6,37 +6,62 @@ a {
   text-decoration: underline;
   cursor: pointer;
 
+  &:hover {
+    @include theme('background-color', 'color-primary--lighten-4', 'link-background-hover');
+    @include theme('color', 'color-primary--darken-2', 'link-color-hover');
+  }
+
   &:focus {
     outline: thin dotted;
   }
 
-  //
-  // @TODO: Add hover state (waiting for design specifications.)
-  //
-  // &:hover {}
+  // Different button link types. These are links styled as a button.
+  &.button-link {
+    @include button;
 
-  // @TODO: Add external-link variant (waiting for design specifications.)
-  // &.external-link {
-  //   &::before {
-  //     @extend %icon-base-css;
-  //     @extend %icon-newtab;
+    display: inline-block;
+    line-height: inherit;
 
-  //     display: inline-block;
-  //     margin-right: .25rem;
-  //     text-decoration: none;
-  //   }
-  // }
+    i {
+      line-height: .7rem;
+    }
+
+    &-primary {
```

Comment:
Why not just use button-link in combination with button-primary || button-secondary || button-dark?
Frauke says: DRY

Reply:
Fixed it.